src/agent/src/storage/mod.rs

```diff
@@ -145,6 +151,8 @@ lazy_static! {
         manager.add_handler(DRIVER_SCSI_TYPE, Arc::new(ScsiHandler{})).unwrap();
         manager.add_handler(DRIVER_VIRTIOFS_TYPE, Arc::new(VirtioFsHandler{})).unwrap();
         manager.add_handler(DRIVER_WATCHABLE_BIND_TYPE, Arc::new(BindWatcherHandler{})).unwrap();
+        #[cfg(feature = "dm-verity")]
```

Comment:
Related to my comments on #8484, could you add `dm-verity` to the list of agent features in `announce()` too?